src/Spatial/Euclidean/Vector2D.cs

```diff
@@ -475,6 +475,15 @@ public Vector2D Add(Vector2D v)
             return new Vector2D(X + v.X, Y + v.Y);
         }
 
+        /// <summary>
+        /// Returns a point equivalent to the vector
```

Comment:
The term _equivalent_ can be misleading. I would rather express this a bit more verbosely, like _Returns a point, whose position vector is equal to the current vector_

Reply:
Should the [comment ](https://github.com/mathnet/mathnet-spatial/blob/f6079916f41912ac0dbfc59003bae673a30ae45f/src/Spatial/Euclidean/Vector3D.cs#L526)in Vector3D ToPoint3D() changed as well? I just copy-pasted mine from there...

Reply:
I would, yes, but it (sadly) seems that this project does not get any attention